BeVolt/App/src/main.c

```diff
@@ -93,14 +93,20 @@ void faultCondition(void){
 			// Toggle Current fault LED
 		}
 		
-		if(Voltage_IsSafe() == OVERVOLTAGE){
+		if(!Voltage_IsSafe()){
```

Comment:
Since we're switching on this anyways, this is probably unnecessary, but whatever.